<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CircleWindow</class>
 <widget class="QMainWindow" name="CircleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请在下方绘制 AOE 网络的事件，并确定活动。单击“构建完毕，下一步”确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="judge">
    <property name="geometry">
     <rect>
      <x>609</x>
      <y>480</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>构建完毕，下一步</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undoBtn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>480</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>撤销</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_7" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>281</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>活动操作</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>事件A</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="vertexUInput">
                <property name="font">
                 <font>
                  <family>Cascadia Code</family>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>事件B</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="vertexVInput">
                <property name="font">
                 <font>
                  <family>Cascadia Code</family>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>活动时间</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="weightInput">
                <property name="font">
                 <font>
                  <family>Cascadia Code</family>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="addEdgeButton">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>添加新活动</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeEdgeButton">
          <property name="text">
           <string>删除活动</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="adjListDisplay">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>560</y>
      <width>401</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>530</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>邻接链表</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
